examples/bpi-monolith-test-suite/bpi/components/workgroup/workgroup.ts

```diff
@@ -6,16 +6,24 @@ export class Workgroup {
      name: string;
      participants: BpiSubject[] = [];
      worksteps: Workstep[] = [];
+     workstepids: Set<string> = new Set("");
```

Comment:
let's maybe use camel case here `workstepIds`
also, please add a comment why workstepIds is needed in addition to worksteps, i think it might be useful